bin/iex

```diff
@@ -43,6 +43,14 @@ readlink_f () {
   fi
 }
 
+ERL="-noshell -user Elixir.IEx.CLI"
 SELF=$(readlink_f "$0")
 SCRIPT_PATH=$(dirname "$SELF")
-exec "$SCRIPT_PATH"/elixir --no-halt --erl "-noshell -user Elixir.IEx.CLI" +iex "$@"
+
+if [ -z "$HOME" ]; then
+  :
+else
+  ERL="-kernel shell_history_path \"$HOME/.elixir-history\" $ERL"
+fi
```

Comment:
@josevalim This looks to be working for me (in isolation anyway):

```
@set ERL=-noshell -user Elixir.IEx.CLI
@set DIR=%HOMEDRIVE%%HOMEPATH%
@if not exist %DIR% goto end
@set ERL=-kernel shell_history_path %DIR%\.elixir-history %ERL%
```

Or you can replace the second line with `@set DIR=%APPDATA%` if you prefer.

If you tell me which file you want this snippet in (I am guessing `iex.bat`?) I can send a PR.